README.md

```diff
@@ -15,9 +15,59 @@ Check the [issues](http://github.com/Chewsterchew/DIRCord/issues) to see what I'
 
 ## Setup guide.
 
-1) Create a server on Discord.
+1) Create a server on Discord, copy the *server's ID*, you'll need it for later.
 
-2) more soon lol
+2) Copy your *user ID*, you'll need it for later.
+
+3) Go to the [Discord Developer Portal](https://discordapp.com/developers/applications/) Make a new application, give it a name, and an avatar if you want.  Copy the *Client ID*, you'll need it for later.  Then head to the **Bot** Section.  Create a bot, scroll down and select the **Administrator** Permission.  Make sure you unselect the slider that makes it a public bot, you don't want anyone but you to have this bot.  Hit the **Copy** button underneath where it says *Bot token*, you'll need this for later.  **DO NOT UNDER ANY CIRCUMSTANCES GIVE *ANYONE* YOUR BOT TOKEN! THIS ALLOWS ANYONE TO TAKE CONTROL OF *YOUR* BOT!**  If you do accidently leak your bot token, do not fear, all hope is not lost.  Simply hit the *Regenerate* button and copy your new token.  **Keep in mind that in order to give it the *Administrator* permission, you *MUST* have 2FA enabled on your discord account.  If you don't know how to do this, go [here](https://support.discordapp.com/hc/en-us/articles/219576828-Setting-up-Two-Factor-Authentication).**
```

Comment:
**Keep in mind that in order to give it the *Administrator* permission, you *MUST* have 2FA enabled on your discord account.
what?? since when?? Highly doubt you need this

Reply:
**DO NOT UNDER ANY CIRCUMSTANCES GIVE *ANYONE* YOUR BOT TOKEN! THIS ALLOWS ANYONE TO TAKE CONTROL OF *YOUR* BOT!** 
Unnecessary.

Reply:
Make a new application, give it a name, and an avatar if you want.  
Should be
Make a new application, give it a name and an avatar if you want.